modules/fleet-membership/membership.tf

```diff
@@ -21,6 +21,7 @@ resource "google_gke_hub_membership" "primary" {
 
   project       = local.hub_project_id
   membership_id = local.gke_hub_membership_name
+  location      = var.membership_location
```

Comment:
Do we need to bump tpb/b version for this?

Reply:
Yup looks like we are past 5.1.0 
https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#510-oct-9-2023

Reply:
Yeah - https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1824 already bumped past what this required